x-pack/plugins/reporting/server/browsers/install.ts

```diff
@@ -19,37 +23,55 @@ const chmod = promisify(fs.chmod);
 interface Package {
   platforms: string[];
 }
-interface PathResponse {
-  binaryPath: string;
-}
 
 /**
  * "install" a browser by type into installs path by extracting the downloaded
  * archive. If there is an error extracting the archive an `ExtractError` is thrown
  */
-export async function installBrowser(
```

Comment:
I switched this from a function to a class, and moved the "install browser" logic from `reporting/server/browsers/create_browser_driver_factory` to here.